solr/solr-ref-guide/src/monitoring-solr-with-prometheus-and-grafana.adoc

```diff
@@ -108,6 +108,38 @@ The number of seconds between collecting metrics from Solr. The `solr-exporter`
 
 The Solr's metrics exposed by `solr-exporter` can be seen at: `\http://localhost:9983/solr/admin/metrics`.
 
+=== Getting metrics from a secure Solr(Cloud)
+
+Your Solr(Cloud) might be secured by measures described in <<securing-solr.adoc#securing-solr,Securing Solr>>. The security configuration can be injected into `solr-exporter` using environment variables in a fashion similar to other clients using <<using-solrj.adoc#using-solrj,SolrJ>>. This is possible because the main script picks up two external environment variables and passes them on to the Java process:
+
+* `JAVA_OPTS` allows to add extra JVM options
+* `CLASSPATH_PREFIX` allows to add extra libraries
+
+Example for a SolrCloud instance secured by
+
+* <<basic-authentication-plugin.adoc#basic-authentication-plugin,Basic Authentication>>
+* <<enabling-ssl.adoc#enabling-ssl,SSL>>
+* <<zookeeper-access-control.adoc#zookeeper-access-control,ZooKeeper Access Control>>
+
+Suppose you have a file `basicauth.properties` with the Solr Basic-Auth credentials:
+
+----
+httpBasicAuthUser=myUser
+httpBasicAuthPassword=myPassword
+----
+
+Then you can start the Exporter as follows (Linux).
+
+[source,bash]
+----
+$ cd contrib/prometheus-exporter
+$ export JAVA_OPTS="-Djavax.net.ssl.trustStore=truststore.jks -Djavax.net.ssl.trustStorePassword=truststorePassword -Dsolr.httpclient.builder.factory=org.apache.solr.client.solrj.impl.PreemptiveBasicAuthClientBuilderFactory -Dsolr.httpclient.config=basicauth.properties -DzkCredentialsProvider=org.apache.solr.common.cloud.VMParamsSingleSetCredentialsDigestZkCredentialsProvider -DzkDigestUsername=readonly-user -DzkDigestPassword=zkUserPassword"
```

Comment:
How would this long line render in HTML ref guide and in PDF ref guide? Can you try to build the ref-guide

    cd solr-refguide && ant dist

and then QA how it looks. An alternative is to split the lines ourselves, e.g.:

    $ export JAVA_OPTS="-Djavax.net.ssl.trustStore=truststore.jks \
    -Djavax.net.ssl.trustStorePassword=truststorePassword \
    -Dsolr.httpclient.builder.factory=org.apache.solr.client.solrj.impl.PreemptiveBasicAuthClientBuilderFactory \
    ....

Reply:
I thought about this. Unfortunately some parameters alone are longer than one line. As the rendering has to break some lines anyway, I didn't break them manually.
In the PDF export it looks ok to me (see attached screenshot). I can't build the HTML export in my current environment.
What do you suggest?

![Doc-Linebreaks](https://user-images.githubusercontent.com/9478787/57587112-ef319f80-74ff-11e9-9325-cc387db122f7.png)